core/interfaces/i_selectable.js

```diff
@@ -11,7 +11,8 @@
 
 'use strict';
 
-goog.provide('Blockly.ISelectable');
+goog.module('Blockly.ISelectable');
+goog.module.declareLegacyNamespace();
 
 goog.requireType('Blockly.IDeletable');
 goog.requireType('Blockly.IMovable');
```

Comment:
Uh oh.  These should have been converted to named requires—and I should have noticed this when originally reviewing this PR.  Ooops.